resources/views/home.blade.php

```diff
@@ -111,6 +111,48 @@
                     @endif
                 @endforeach
 
+                <!-- Workshop secretaries -->
+                <h5><a id="workshop-administrator-collapsible" href="javascript:void(0)" onclick="toggleCollContent()">
+                    <span class="dropdown-triangle">▶</span>
+                    @lang('role.workshop-functionaries')
+                </a></h5>
+                <div id="workshop-administrator-content" style="display: none">
+                    <ul>
+                        @foreach(\App\Models\Workshop::all() as $workshop)
+                        <li>
+                            <b>{{$workshop->name}}</b>
+                            <ul>
+                                <li>@lang('role.'.\App\Models\Role::WORKSHOP_LEADER):
+                                    <i>
+                                        {{
+                                            implode(
+                                                ', ',
+                                                array_map(
+                                                    function ($admin) {return $admin->name;},
+                                                    $workshop->leaders()->all()
```

Comment:
$workshop->leaders->->pluck('name')->implode(', ');

Reply:
Or it's better to calculate them in the backend and add it to the $contacts (see above), mapped to the workshops.